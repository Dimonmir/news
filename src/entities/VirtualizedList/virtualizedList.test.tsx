import { render } from '@testing-library/react';
import { VirtualizedList } from './index';

test('VirtualizedList snapshot', () => {
  const { asFragment } = render(
    <VirtualizedList
      data={[
        {
          API: 'AdoptAPet',
          Description: 'Resource to help get pets adopted',
          Auth: 'apiKey',
          HTTPS: true,
          Cors: 'yes',
          Link: 'https://www.adoptapet.com/public/apis/pet_list.html',
          Category: 'Animals',
          Img: 'background1.jpg',
        },
        {
          API: 'Axolotl',
          Description: 'Collection of axolotl pictures and facts',
          Auth: '',
          HTTPS: true,
          Cors: 'no',
          Link: 'https://theaxolotlapi.netlify.app/',
          Category: 'Animals',
          Img: 'background2.jpg',
        },
      ]}
      onDelete={() => {}}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
