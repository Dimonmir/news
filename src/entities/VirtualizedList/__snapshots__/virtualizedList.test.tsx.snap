// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VirtualizedList snapshot 1`] = `
<DocumentFragment>
  <div
    style="height: 600px; overflow: auto; position: relative;"
  >
    <div
      class="css-1x6szdv"
      style="height: 800px;"
    >
      <div
        style="position: absolute; top: 0px; left: 25%; transform: translateY(0px); height: 400px; padding: 6px 12px; width: 50%;"
      >
        <div
          class="css-139pbvv"
        >
          <div
            class="card__header"
          >
            <div
              class="card__flexBox"
            >
              <span
                class="card__title"
              >
                API:
              </span>
               
              <div>
                AdoptAPet
              </div>
            </div>
            <button
              aria-label="delete"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="DeleteIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <img
            alt="photo"
            src="background1.jpg"
          />
          <div
            class="card__desc"
          >
            Resource to help get pets adopted
          </div>
          <div
            class="card__flexBox"
          >
            <span
              class="card__title"
            >
              Categories:
            </span>
             
            <div>
              Animals
            </div>
          </div>
          <div
            class="card__flexBox"
          >
            <span
              class="card__title"
            >
              Link:
            </span>
             
            <a
              href="https://www.adoptapet.com/public/apis/pet_list.html"
            >
              https://www.adoptapet.com/public/apis/pet_list.html
            </a>
          </div>
        </div>
      </div>
      <div
        style="position: absolute; top: 0px; left: 25%; transform: translateY(400px); height: 400px; padding: 6px 12px; width: 50%;"
      >
        <div
          class="css-139pbvv"
        >
          <div
            class="card__header"
          >
            <div
              class="card__flexBox"
            >
              <span
                class="card__title"
              >
                API:
              </span>
               
              <div>
                Axolotl
              </div>
            </div>
            <button
              aria-label="delete"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="DeleteIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <img
            alt="photo"
            src="background2.jpg"
          />
          <div
            class="card__desc"
          >
            Collection of axolotl pictures and facts
          </div>
          <div
            class="card__flexBox"
          >
            <span
              class="card__title"
            >
              Categories:
            </span>
             
            <div>
              Animals
            </div>
          </div>
          <div
            class="card__flexBox"
          >
            <span
              class="card__title"
            >
              Link:
            </span>
             
            <a
              href="https://theaxolotlapi.netlify.app/"
            >
              https://theaxolotlapi.netlify.app/
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
